### ISSUE TITLE
There is a C# interactive, contradicting an article

### ISSUE THREAD

**Issue by OptimisticPeach** [swlaschin/fsharpforfunandprofit.com#30]
Hi! I was just reading [this](https://fsharpforfunandprofit.com/posts/fvsc-sum-of-squares/) page, and near the bottom the following is said:

> F# has an interactive window where you can test the code immediately and play around with it. In C# there is no easy way to do this.

Just like how F# has an interactive REPL, C# has one too, which works just fine:
```
View -> Other Windows -> C# Interactive
```

**Comment by swlaschin** [comment]
Yes, that page is over 7 years old now and very out of date.  Thanks for the reminder to update it!
